web/src/common/theme.js

```diff
@@ -52,6 +52,7 @@ const theme = responsiveFontSizes(
       icon: {
         lightPurple: basePalette.lightPurple,
       },
+      ...basePalette,
```

Comment:
I was hoping to avoid putting this in the actual theme object. This way we can enforce that the property has some semantic meaning for better readability and obvious re-usability

Reply:
moved